docs/en/build-store/refresh-control.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Validations and refresh control
+table_of_contents: true
+---
+
+# Validations and refresh control
+
+## What is refresh control?
+
+_“🎵 baby don't update me, don't update me no more 🎵”_
+
+
+Refresh control is a mechanism for controlling the revisions of a snap that will be offered as updates  to a device. A *gating snap* can control the revisions of a *gated snap* so that, if the *gating snap* is installed on a device, only revisions of the *gated snap* which have been validated by the publisher of the *gating snap* can be installed or refreshed to.
+
+A gating snap can control revisions of one or more gated snaps. Similarly, a gated snap can require verification from more than one gating snap, and a particular revision of the gated snap will only be installable or refreshable to, if it satisfies the constraints of all its gating snaps. However, because it can be confusing to device operators, use of multiple gating snaps to control one specific gated snap is not recommended. The best practice is to use a single gating snap for each gated snap.
```

Comment:
Maybe name this the "gating snaps's refresh control scheme". We could say: "A snap can be made a gating snap by declaring the gated snaps it controls. This is the gating snap's _refresh control scheme_.  This is configured by a Reviewer in the store in which the gating snap is published. Then, the gating snap's publisher can sign and issue __validation__ assertions that specify the revisions gated snaps and held to."

Reply:
Style and naming - Feel free to change style and phrasing post-landing. For "refresh control scheme" I hesitate to introduce new terminology without wider discussion. If we do have that discussion this can be updated later in canonical-docs.